.github/workflows/continuous-integration-workflow.yml

```diff
@@ -118,185 +118,6 @@ jobs:
           name: logs
           path: ./tools/docker/logs
 
-  solidity-lint:
```

Comment:
maybe rename this file to core-workflow.yml?

Reply:
I think it still might be good to include the paths directive here too, for example if you change the docs/README/Makefile we don't want to run the tests

Reply:
nice idea, yeah